app/src/main/java/swati4star/createpdf/activity/WelcomeActivity.java

```diff
@@ -108,9 +108,9 @@ public void onPageScrollStateChanged(int arg0) {
     };
 
 
-    /**
-     * View pager adapter
-     */
+    
+     // View pager adapter
```

Comment:
why?

Reply:
There are no multiple lines here to use /*....*/ so preferring a single line comment here as observed the same above in the code.

Reply:
Even if it is single line, we prefer javadoc style comments: http://drjava.org/docs/user/ch10.html

Reply:
I mean /* .. */ in my above comment.

Reply:
If you wish to make the formatting uniform, please do it for whole file.
and don't make /* .. */  to //
We prefer javadoc style comments structure (* .. */ )

Reply:
Sure